wrapper/godot-library/src/main/kotlin/godot/core/Dictionary.kt

```diff
@@ -53,16 +53,25 @@ class Dictionary: CoreType {
 
     fun erase(key: Variant) = godot_dictionary_erase(nativeValue, key.nativeValue)
 
+    fun erase(key: Any) = this.erase(Variant.from(key))
+
     fun has(key: Variant): Boolean = godot_dictionary_has(nativeValue, key.nativeValue)
 
-    fun get(key: Variant): Variant? = godot_dictionary_operator_index(nativeValue, key.nativeValue)?.pointed?.readValue()?.let { Variant(it) }
+    fun has(key: Any): Boolean = this.has(Variant.from(key))
+
+    operator fun get(key: Variant): Variant? = godot_dictionary_operator_index(nativeValue, key.nativeValue)?.pointed?.readValue()?.let { Variant(it) }
+
+    operator fun get(key: Any): Variant? = this[Variant.from(key)]
 
+    operator fun set(key: Variant, value: Variant) = godot_dictionary_set(nativeValue, key.nativeValue, value.nativeValue)
+
+    operator fun set(key: Any, value: Any) = this.set(Variant.from(key), Variant.from(value))
```

Comment:
Wouldn't this be better to use CoreType instead of Any ?

Reply:
No, because we had to cast kotlin's primitive too
![image](https://user-images.githubusercontent.com/48322356/74740598-78c68880-525b-11ea-9501-d6ede0b1c310.png)